core/src/com/unciv/ui/overviewscreen/NotificationsOverviewTable.kt

```diff
@@ -88,7 +88,7 @@ class NotificationsOverviewTable(
             for (notification in categoryNotifications) {
                 val notificationTable = Table(BaseScreen.skin)
 
-                val labelWidth = maxEntryWidth * notification.icons.size - 10f
+                val labelWidth = maxEntryWidth - iconSize * notification.icons.size - 10f
```

Comment:
Are you sure you're not missing () here?

Reply:
Do you mean parentheses as in for order of operations?